store/tikv/client_batch.go

```diff
@@ -424,12 +426,16 @@ func sendBatchRequest(
 	}
 	ctx1, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
+retry:
 	select {
 	case batchConn.batchCommandsCh <- entry:
```

Comment:
it has the chance some "requests" be inserted into `addr`'s `batchConn.batchCommandsCh` after `c.eventCh` have msg but before receive and handle `c.eventCh`, how about those "requests" will be handled?

Reply:
`failPendingRequests` will handle the pending requests

Reply:
In `batchSendLoop`, the loop exit condition is `fetchAllPendingRequests` gets nothing, so there won't be a case that `fetchAllPendingRequests` get some request and exit (ignore them).